taotie/tests/test_utils.py

```diff
@@ -15,6 +15,14 @@
 def test_get_datetime(input, expected):
     assert get_datetime(input) == expected
 
+def test_load_dotenv():
+    with patch.dict('os.environ', {'TEST_KEY': 'test value'}):
+        load_dotenv()
+        assert os.environ.get('TEST_KEY') == 'test value'
```

Comment:
This failed the black check. See the error:
```
Run black --check .
would reformat /home/runner/work/taotie/taotie/taotie/tests/test_utils.py
```

You would need to check the black check.